presentation/src/main/java/com/maxrzhe/presentation/viewmodel/impl/contacts/ContactDetailViewModel.kt

```diff
@@ -169,9 +173,10 @@ class ContactDetailViewModel internal constructor(
             addContactUseCase.execute(contact)
                 .collect {
                     if (it is Resource.Success) {
```

Comment:
try to change `if-else` to `when`. below the same

Reply:
fixed